src/dev/impl/DevToys/DevToys.csproj

```diff
@@ -447,7 +512,54 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
-    <PRIResource Include="Strings\*\*.resw" />
```

Comment:
Remove all these `PRIResource` and bring back `<PRIResource Include="Strings\*\*.resw" />` please :)